plugins/scanners/fossid/src/test/kotlin/FossIdNamingProviderTest.kt

```diff
@@ -0,0 +1,82 @@
+package org.ossreviewtoolkit.plugins.scanners.fossid
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.extensions.clock.TestClock
+import io.kotest.matchers.equals.shouldBeEqual
+import io.kotest.matchers.ints.shouldBeLessThanOrEqual
+
+import io.mockk.every
+import io.mockk.mockkStatic
+
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.ZoneOffset
+
+class FossIdNamingProviderTest : WordSpec({
+
+    "createScanCode" should {
```

Comment:
> instead of pulling `TestClock`

Good idea. I also felt uneasy about introducing the `kotestExtensionsClock`.

Reply:
Sure I can mock LocalDateTime. Just kotest lib looked a bit better to me.

Reply:
I believe it's better to use fakes instead of mocks. For example, 
constructor param `private val clock: Clock = Clock.systemUTC()` could be added, and the
test can inject a fake implementation. 

However, to me it's not clear yet why the time is needed at all. Can you explain this please?

Reply:
As FossID scan code contains time-based values, to test generated scan code against expected value, there is urge to generate such code with known time of generation or mock time retrieval method.

Reply:
Clock is removed now. LocalDateTime.now() is mocked, and works just fine.